fury/window.py

```diff
@@ -740,8 +740,9 @@ def play_events_from_file(self, filename):
 
     def add_window_callback(self, win_callback, event=Command.ModifiedEvent):
         """Add window callbacks."""
-        self.window.AddObserver(event, win_callback)
+        window_id = self.window.AddObserver(event, win_callback)
         self.window.Render()
+        return window_id
```

Comment:
This change was made to assure anyone working with the window callback or the interactor callback could have access to the added callback id, in case they need to remove or change it later. I use it, for example, to remove the callbacks added by an effect inside the `remove_effect()` method